autogen/agentchat/groupchat.py

```diff
@@ -178,6 +180,15 @@ def run_chat(
                     raise
             if reply is None:
                 break
+
+            if (
+                type(reply) == dict
+                and self._is_termination_msg(reply)
+                or type(reply) == str
+                and self._is_termination_msg({"content": reply})
+            ):
+                break
+
```

Comment:
@LittleLittleCloud, should these lines be after `speaker.send` ??

Reply:
also, I usually use `isinstance()` instead of `type()`

Reply:
> @LittleLittleCloud, should these lines be after `speaker.send` below??

Did this suggestion make sense?

Reply:
nice catch